src/test/scala/net/ceedubs/ficus/ConfigSerializer.scala

```diff
@@ -45,9 +46,10 @@ object ConfigSerializer {
 
 final case class ConfigSerializerOps[A](a: A, serializer: ConfigSerializer[A]) {
   def asConfigValue: String = serializer.serialize(a)
+  def toConfigValue = ConfigFactory.parseString( s"dummy=$asConfigValue").root().get("dummy")
```

Comment:
can we add the return type here, just so that people don't have to read through the code to tell the difference between `asConfigValue` and `toConfigValue` (also how about change the name to `toConfig` to further differentiate the two?)